debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -69,6 +70,11 @@
 @Immutable
 public class MySqlValueConverters extends JdbcValueConverters {
 
+    @FunctionalInterface
```

Comment:
Moved this here to avoid the dependency from `MySqlValueConverters` to `BinlogReader`.